<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-BP6165A" ignored="0" name="CompleteTest" tests="5" failures="2" timestamp="2020-04-22T18:12:35 IST" time="236.395" errors="0">
  <testcase name="ValicadteWelcomeMsg" time="1.281" classname="com.mylearning.TestCases.LoginTest"/>
  <testcase name="ValidateLogo" time="0.476" classname="com.mylearning.TestCases.LoginTest"/>
  <testcase name="checkbranch1" time="0.003" classname="com.mylearning.TestCases.LoginTest"/>
  <testcase name="testfail" time="0.22" classname="com.mylearning.TestCases.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.mylearning.TestCases.LoginTest.testfail(LoginTest.java:72)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testfail -->
  <testcase name="validatelogin" time="20.818" classname="com.mylearning.TestCases.LoginTest">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	Unhandled exception type IOException
	Unhandled exception type IOException
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
Unhandled exception type IOException
Unhandled exception type IOException

at com.mylearning.TestCases.LoginTest.validatelogin(LoginTest.java:57)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validatelogin -->
</testsuite> <!-- CompleteTest -->
